### ISSUE TITLE
Restart timer from scorekeeper webpage without too much interference with the score entry

### ISSUE THREAD

**Issue by alanggreen** [FirstLegoLeague/fllscoring#246]
In our competitions the scorekeeper often is the person that starts the timer. While they are waiting for the MC to do the countdown, they cannot be on the score entry screen, and this slows down their work.

Need a way to start / stop the timer from score entry page (maybe with a hot-key)

**Comment by Jheronymus** [comment]
I like the hot-key idea. Since all these things are controlled via message bus, maybe it is a good idea to make some kind of hotkey module that acts the same across all of the solutions (so hotkeys are the same for display system and fllscoring and so on). Therefor the apps remain separate but can still be controlled from one or the other.

**Comment by alanggreen** [comment]
Yes, that's a good idea, because then from the UX side, it all looks like one app.

**Comment by rikkertkoppes** [comment]
We do have hotkeys for our clock application already. Also for display system it is possible to configure hotkeys. Extending it to scoring makes sense.

Not however that on touch devices hotkeys do not make much sense, so there must also be a way to do it via the ui.

Anyone for voice commands? I wouldn't pick LEGO as the trigger word though...

**Comment by Jheronymus** [comment]
There is an open issue regarding the controls for the clock: https://github.com/FirstLegoLeague/clock/issues/18 This should be resolved first (I'm actually not sure if this the same thing as I experienced at our Nationals, will confirm with Rob/Kenny)

We could even visualize these hotkeys like a configurable toolbar that is the same for all apps.
